### ISSUE TITLE
How to get an id when no block is defined?

### ISSUE THREAD

**Issue by pheyvaer** [rubensworks/graphql-to-sparql.js#31]
Assume you have the following query

```
{
  name
}
```
how do you select only the entities with id `http://example.com/john`?
When [setting a block-scoped id](https://github.com/rubensworks/graphql-to-sparql.js#setting-a-block-scoped-id) there is like a "parent" where you can set the id (`hero` in the example of the link), but what do you need to do if you don't have that?

**Comment by rubensworks** [comment]
Something like this *should* work:
```
{
  id(_:'http://example.com/john')
  name
}
```

If it doesn't, then that should definitely become a feature :-)

**Comment by pheyvaer** [comment]
If I try that then I get `GraphQLError: Syntax Error: Unexpected single quote character ('), did you mean to use a double quote (")?`.

**Comment by rubensworks** [comment]
And does the double-quoted version work?

**Comment by pheyvaer** [comment]
In that case, no error is thrown, but no results are returned.

**Comment by julianrojas87** [comment]
I tried something this scenario but instead of passing the id parameter on the query, I defined it in the `@context` and it worked fine:
```js
{
   "@context": { "param": "http://example.org/john" }
}
----------------------------------------------------------------------
{
   id(_:param)
   name
}
```

**Comment by rubensworks** [comment]
Right, I forgot about that even. This makes `param` a GraphQL enum, which are always seen as URIs. Thanks for sharing @julianrojas87.

In any case, the `id` field will always point to a URI, so it should be possible to use a string value as an IRI instead of a literal. This is in fact not a bug, but a not-yet-implemented feature.
